src/common/components/FileInput/FileInput.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Center,
+  Flex,
+  FlexProps,
+  FormControl,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputProps,
+  Text,
+  VStack,
+  Image,
+} from '@chakra-ui/react';
+
+import { ImageIcon } from '../Icons';
+
+interface FileInputProps extends InputProps {
+  label: string;
+  selectedFile: File;
+  preview: string;
+  errorMessage?: string;
+  flexProps?: FlexProps;
+}
+
+export const FileInput = ({
```

Comment:
I think this component mainly caters for Image files, so can we rename to `ImageFileInput`? the Input component accepts `image/*`